README.yaml

```diff
@@ -85,6 +85,18 @@ description: |-
 usage: |-
   Using a [canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html).
 
+  NOTE: As of 2.0.2, this module will now require the `awsutils` provider to be defined with a region. This can be the same region as the `aws` provider.
+
+  ```hcl
+  provider "awsutils" {
+    region = var.region
+  }
+
+  provider "awsutils" {
```

Comment:
Also worth mentioning `awsutils` has to be included in `required_providers` ?

Reply:
Should it be included in `required_providers` in this module? If so, `no-release` PR tag should be removed.

Reply:
Once region is optional in the awsutils provider (like it is for the aws provider) then the provider won't need to be explicitly set, no?